docs/USAGE.md

```diff
@@ -6,22 +6,33 @@ This document describes available Python APIs for **OpenVINO™ integration with
 
 ## APIs for essential functionality
 
-To add the **OpenVINO™ integration with TensorFlow** package to your TensorFlow python application, import the package using this line of code:
+To add the environment variable package to your TensorFlow python application, import the package using this line of code:
 
```

Comment:
Didn't follow "environment variable package" change. The previous description of "OpenVINO™ integration with TensorFlow package" looks more intuitive to me.

Reply:
Thanks for pointing out, will roll back this change.